backend/queues/printfulSyncProcessor.js

```diff
@@ -1,16 +1,43 @@
 const queues = require('./queues')
 
-const downloadProductData = require('../scripts/printful/downloadProductData')
-const writeProductData = require('../scripts/printful/writeProductData')
-const downloadPrintfulMockups = require('../scripts/printful/downloadPrintfulMockups')
-const resizePrintfulMockups = require('../scripts/printful/resizePrintfulMockups')
+const downloadProductData = require('../logic/printful/sync/downloadProductData')
+const writeProductData = require('../logic/printful/sync/writeProductData')
+const downloadPrintfulMockups = require('../logic/printful/sync/downloadPrintfulMockups')
+const resizePrintfulMockups = require('../logic/printful/sync/resizePrintfulMockups')
 
 const { Shop } = require('../models')
 
 const attachToQueue = () => {
   const queue = queues['printfulSyncQueue']
   queue.process(processor)
   queue.resume() // Start if paused
+
+  const updateSyncStatus = async (job, status, hasChanges) => {
+    if (!job.data) return
+
+    const { shopId } = job.data
+
+    if (!shopId) return
+
+    await Shop.update(
```

Comment:
ah I did not realize sequelize does not update a column if the value is undefined... for code readability purposes,  perhaps add a comment to mention this explicitly, or do not pass hasChanges to the update method if it's undefined.

Reply:
Have made this change :)